onnxruntime/core/framework/execution_frame.cc

```diff
@@ -583,7 +583,7 @@ static Status AllocateTensorSequence(OrtValue& ort_value) {
   return Status::OK();
 }
 
-static Status AllocateSparseTensor(MLValue& mlvalue, const DataTypeImpl& ml_type, AllocatorPtr allocator,
+static Status AllocateSparseTensor(OrtValue& mlvalue, const DataTypeImpl& ml_type, AllocatorPtr allocator,
```

Comment:
Since you've made this change in a number of place, should we also rename the file ml_value.h to ort_value.h? The filename doesn't match the class it houses. Can be done in a separate PR.

Reply:
Sure. I will do it in another PR.